### PR TITLE
Book Store: Styling

### PR DESCRIPTION
**Pull request by binodbhusal**
**The following tasks were accomplished in this milestone :** 

- Added CSS styling to bookstore 
- Added CSS modules

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -11,14 +12,45 @@ const BookItem = ({ book }) => {
...
+            <div className={styles.btncontainer}>
+              <button className={styles.btncol} type="button">
+                Comments
+                <span className={styles.vLine}>|</span>
+              </button>
+              <button className={styles.btncol} type="button" onClick={handleRemove}>
+                Remove
+                <span className={styles.vLine}>|</span>
+              </button>
+              <button className={styles.btncol} type="button">Edit</button>
```
Comment (Tufahel) [inline src/components/BookItem.js:30]: - [ ] I can see the `text color` of this section is not matching with the given design guideline. Kindly implement the given color for the specific section. Such as for the texts the color should be `#4386bf`.

File: src/components/Form.js [hunk 2]
```diff
@@ -16,16 +17,17 @@ const Addform = () => {
   };
   return (
     <>
-      <h2>Add New Book</h2>
-      <form onSubmit={addNewBook}>
-        <input type="text" name="bookTitle" placeholder="Booktitle" />
-        <input type="text" name="author" placeholder="Author" />
-        <select name="category">
+      <h3 className={styles.headerform}>ADD NEW BOOK</h3>
```
Comment (Tufahel) [inline src/components/Form.js:20]: - [ ] Same here. The color of the text is not correct as per the given design. It should be `#888888`. Kindly fix this issue.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @binodbhusal :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Follow the correct Git flow.
✔️ Application is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.js (hunk 1 above)
Comment (peterrobert) [reply src/components/BookItem.js:30]: This change has been done well

File: src/components/Form.js (hunk 2 above)
Comment (peterrobert) [reply src/components/Form.js:20]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @binodbhusal, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**